data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -4,9 +4,17 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
     <changeSet author="R3.Corda" id="crypto-creation-v1.0">
         <createTable tableName="crypto_wrapping_key">
+            <column name="id" type="VARCHAR(64)">
```

Comment:
```suggestion
            <column name="id" type="UUID">
```

Reply:
Liquibase docs are a bit limited, so I'm not sure what that would mean. My expectation is that some of the generation values are user supplied, for root wrapping keys, and that would mean the user configuration would have to be acceptable to the database.

https://xenovation.com/blog/development/java/java-professional-developer/liquibase-related-sql-java-types appears to show that UUID is mapped differently across databases.

Reply:
It's fine. That's how we do it elsewhere. There was an issue with this in old versions of Liquibase.